### PR TITLE
Refactor forms to use Final Form. Part of UIIN-668. 

### PR DESCRIPTION
**Pull request by mkuklis**
https://issues.folio.org/browse/UIIN-668

### CODE REVIEW

**Comment by mkuklis** [comment]
@zburke I wonder if you could give it a try and see it things work on your end. The BigTest tests are passing for me and I spent some time just testing it manually.

**Comment by mkuklis** [comment]
Dear Zak Burke,

I cannot express my gratitude enough for your exceptional testing skills and extraordinary care related to this issue.

With Love,
Michał

File: package.json [hunk 1]
```diff
@@ -388,20 +388,21 @@
...
+    "react-final-form-arrays": "^3.1.0",
     "react-hot-loader": "^4.3.12",
     "react-intl": "^2.3.0",
     "react-router": "^4.0.0",
-    "react-router-dom": "^4.0.0",
-    "redux-form": "^7.0.3"
+    "react-router-dom": "^4.0.0"
   },
   "peerDependencies": {
     "@folio/stripes": "^2.1.0",
```
Comment (zburke) [inline package.json]: To make sure this include a `stripes` release with `stripes-final-form`, this should be bumped to 
```
    "@folio/stripes": "^2.9.0",
```

Comment (mkuklis) [reply package.json]: Thanks @zburke. I just fixed it. I'm going to merge this in. Let's see if this actually works.

**Comment by mkuklis** [comment]
https://issues.folio.org/browse/UIU-1141

**Pull request merged**